Dockerfile.builder.tmpl

```diff
@@ -1,8 +1,6 @@
 {{ .base | strings.TrimSpace }}
 
-RUN apk add --no-cache \
-    git \
-    ca-certificates
```

Comment:
I think I'd like to keep this as still one-package-per-line, for readability's sake.

Reply:
Should I change the main image one to match then? I just thought there should be a common style between them.

Reply:
I guess so.

Thanks!

Reply:
No problem, done!